scrapy/extensions/logstats.py

```diff
@@ -30,21 +30,25 @@ def from_crawler(cls, crawler):
     def spider_opened(self, spider):
         self.pagesprev = 0
         self.itemsprev = 0
-
+        self.responsebytesprev = 0.0
         self.task = task.LoopingCall(self.log, spider)
         self.task.start(self.interval)
 
     def log(self, spider):
         items = self.stats.get_value('item_scraped_count', 0)
         pages = self.stats.get_value('response_received_count', 0)
+        response_bytes = self.stats.get_value('downloader/response_bytes', 0.0) / 1000
```

Comment:
I wonder if maybe we should show bytes instead of kilobytes. I mean, given the stats are in bytes, and that both pages and items are integers, it seems more consistent to me to use the byte count in the message as well.